src/Faker/Provider/Miscellaneous.php

```diff
@@ -198,6 +198,7 @@ class Miscellaneous extends Base
         'wal_ET', 'wo_SN', 'xh_ZA',
         'yo_NG', 'zh_CN', 'zh_HK',
         'zh_MO', 'zh_SG', 'zh_TW', 'zu_ZA',
+        'ps_AF'
```

Comment:
This needs to be put on Alphabet please

Reply:
How About I just remove it completely as it is there in this line https://github.com/fzaninotto/Faker/blob/27d87cb42f5608d0f41e6f6d5f835e3b70d6888f/src/Faker/Provider/Miscellaneous.php#L178